src/library/dlgtrackinfo.cpp

```diff
@@ -171,18 +171,22 @@ void DlgTrackInfo::init() {
             &TapFilter::tapped,
             this,
             &DlgTrackInfo::slotBpmTap);
-
+    // Swap Title - Artist
+    connect(btnSwapArtistTitle,
+            &QPushButton::clicked,
+            this,
+            &DlgTrackInfo::slotSwapArtistTitle);
     // Metadata fields
     connect(txtTitle,
-            &QLineEdit::editingFinished,
+            &QLineEdit::textChanged,
             this,
             [this]() {
                 txtTitle->setText(txtTitle->text().trimmed());
                 m_trackRecord.refMetadata().refTrackInfo().setTitle(
                         txtTitle->text());
             });
     connect(txtArtist,
-            &QLineEdit::editingFinished,
+            &QLineEdit::textChanged,
```

Comment:
This means we now write to TrackRecord  for every change.

Why not just copy the `m_trackRecord` calls to `slotSwapArtistTitle()`?